python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -87,6 +87,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         dtype=bool,
         default=True
     )
+    astrometer = pexConfig.ConfigurableField(
+        target = measAstrom.AstrometryTask,
+        doc = "astrometry task; used to match sources to reference objects, but not to fit a WCS",
+    )
```

Comment:
Do recall what class that was?  I think now obs_subaru is now integrated into the stack enough we should at least just consider changing it here.